packages/salesforcedx-vscode-lwc/test/vscode-integration/activation/MockExtensionContext.ts

```diff
@@ -8,9 +8,11 @@ import * as path from 'path';
 import {
   EnvironmentVariableCollection,
   EnvironmentVariableMutator,
+  Extension,
```

Comment:
update for new vscode types version.

Reply:
Is there a doc on this one I can read up on?

Reply:
Nope, but I'll add it to the list of things to cover in the L&L